chapters/synchronous.tex

```diff
@@ -807,10 +807,18 @@ \subsection{Sub-clock conversion operators}\doublelabel{sub-clock-conversion-ope
 \\ \hline
 \textbf{noClock}(u)
 &
-The clock of y = \textbf{noClock}(u) is always inferred. At every tick
-of the clock of y, the operator returns the value of u from the last
-tick of the clock of u. If \textbf{noClock}(u) is called before the
-first tick of the clock of u, the start value of u is returned.\\ \hline
+The operator \textbf{noClock} is meant to enable efficient data exchange 
+between different Base-Clocks. The clock of y = \textbf{noClock}(u) is 
+always inferred. At every tick of the clock of y, the operator returns 
+the value of u from the last tick of the clock of u. If \textbf{noClock}(u) 
+is called before the first tick of the clock of u, the start value of u is 
+returned.
+{[}\emph{Note, it is necessary that \textbf{noClock} returns the value from 
+the last tick of the clock of u, as between different Base-Clocks there is
+no guaranteed execution order. If \textbf{noClock} is used between different 
+Sub-Clocks of the same Base-Clock it is equivalent to \textbf{sample}(\textbf{previous}(u)).}
```

Comment:
`sample(...) `is not allowed to have a clocked expression as input. It should be `subSample(previous(u),1)`.

Reply:
I believe you mean `subSample(previous(u))` since they are two different sub-clock partitions and thus we need to infer the sub-sampling factor. However, it can also handle other cases (including superSample etc); so we need a more general statement.